dds_web/commands.py

```diff
@@ -1145,9 +1145,6 @@ def collect_stats():
     # Get current time
     current_time = dds_web.utils.timestamp(ts_format="%Y-%m-%d")
 
-    # Get the instance name (DEVELOPMENT, PRODUCTION, etc.)
```

Comment:
Sorry, I was unclear. What I meant with my previous comment about this not being used, is that we _should_ use the variable somewhere, like we do in the other command. In this case we should change the `error_subject` to include this. It should become: `[CRONJOB] (DEVELOPMENT) Error during collection of DDS unit- and user statistics.` for example.

Reply:
Oos Yeah it makes sense 🤦